client/layout/index.jsx

```diff
@@ -144,6 +145,17 @@ Layout = React.createClass( {
 		);
 	},
 
+	renderPreview() {
+		if ( config.isEnabled( 'preview-layout' ) && this.props.section.group === 'sites' ) {
```

Comment:
Was going to ask whether it made more sense to move the first condition (config check) into `DesignPreview` proper, letting the component itself decide whether it's enabled.

That'd would also make the remainder condition (check that group === sites) smaller and maybe make it more comfortable to ditch `renderPreview` altogether and have the thing inlined in the `render`, possibly removing indirection. #nit-picks